### PR TITLE
Basic and simple test for sample analysis load

### PR DESCRIPTION
**Pull request by hassanfa**
These new additions include:
- placeholder for analysis build and load
- simple test for analysis sample test

### CODE REVIEW

File: vogue/build/analysis.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+def build_analysis(analysis: dict, analysis_type: str):
+    """build a analysis object"""
+    return analysis
```
Comment (moonso) [inline vogue/build/analysis.py]:
```suggestion
    return analysis

```

File: vogue/load/analysis.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from vogue.adapter import VogueAdapter
+from vogue.build.analysis import build_analysis
+
+def load_cancer_analysis(adapter: VogueAdapter, sample_id: str, analysis: dict):
```
Comment (mayabrandi) [inline vogue/load/analysis.py]:
```suggestion
def load_cancer_analysis(adapter: VogueAdapter, analysis: dict):
```

File: vogue/load/analysis.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    """Load information from a cancer analysis"""
+    sample_obj = adapter.sample(sample_id)
```
Comment (mayabrandi) [inline vogue/load/analysis.py]:
```suggestion
sample_id = analysis.get('lims_sample_id')
    sample_obj = adapter.sample(sample_id)
```

File: tests/load/test_load_analysis.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    ## GIVEN a adapter with a sample
+    
+    assert adapter.sample_collection.insert_one(test_sample)
+
+    assert adapter.analysis_collection.find_one() is None
+    ## WHEN adding a cancer analysis
+    
+    sample_id = test_sample['_id']
+    analysis_obj = load_cancer_analysis(adapter, sample_id, cancer_analysis)
+    print(analysis_obj)
```
[the remark this answers is not in the document]
Comment (hassanfa) [reply tests/load/test_load_analysis.py]: To check the output :-) I'll remove it.

File: tests/load/test_load_analysis.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+
+from vogue.load.analysis import load_cancer_analysis
+
+def test_load_simple_cancer(adapter, test_sample, cancer_analysis):
```
[the remark this answers is not in the document]
Comment (hassanfa) [reply tests/load/test_load_analysis.py]: doesn't apply after next commit, discussed with Maya.

File: vogue/load/analysis.py (hunk 2 above)
Comment (hassanfa) [reply vogue/load/analysis.py]: doesn't apply after next commit, discussed with Maya.

File: vogue/load/analysis.py (hunk 3 above)
Comment (hassanfa) [reply vogue/load/analysis.py]: doesn't apply after next commit, discussed with Maya.

**Pull request merged**